scripts/ci/mlflow-e2e.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+print_bold() {
+    echo
+    echo -e "\033[1m$1\033[0m"
+}
+
+if ! git describe --tags --exact-match &> /dev/null; then
+    print_bold "➤ Build FuseML client:"
+    if [ -d "fuseml-core" ]; then
+        cd fuseml-core
+        git fetch origin
+        git reset --hard origin/main
+        git clean -f -d
+    else
+        git clone https://github.com/fuseml/fuseml-core.git
+        cd fuseml-core
+    fi
+    make deps generate build_client
+    mv bin/fuseml ../fuseml
+    cd ..
+    rm -rf fuseml-core
+fi
+
+
+export FUSEML_SERVER_URL=http://$(kubectl get VirtualService -n fuseml-core fuseml-core -o jsonpath="{.spec.hosts[0]}")
+print_bold "⛓  FuseML URL: ${FUSEML_SERVER_URL}"
+./fuseml version
+
+print_bold "➤ Checkout examples repository:"
+if [ -d "fuseml-examples" ]; then
+    cd fuseml-examples
+    git fetch origin
+    git reset --hard origin/main
+    git clean -f -d
+    cd ..
+else
+    git clone --depth 1 https://github.com/fuseml/examples.git fuseml-examples
+fi
+
+print_bold "➤ Register Codeset:"
+./fuseml codeset register --name "mlflow-wines" --project "mlflow-project-01" "fuseml-examples/codesets/mlflow-wines"
+
+export ACCESS=$(kubectl get secret -n fuseml-workloads mlflow-minio -o json | jq -r '.["data"]["accesskey"]' | base64 -d)
+export SECRET=$(kubectl get secret -n fuseml-workloads mlflow-minio -o json | jq -r '.["data"]["secretkey"]' | base64 -d)
+
+sed -i -e "/AWS_ACCESS_KEY_ID/{N;s/value: [^ \t]*/value: $ACCESS/}" fuseml-examples/workflows/mlflow-sklearn-e2e.yaml
+sed -i -e "/AWS_SECRET_ACCESS_KEY/{N;s/value: [^ \t]*/value: $SECRET/}" fuseml-examples/workflows/mlflow-sklearn-e2e.yaml
+
+print_bold "➤ Create Workflow (if needed):"
```

Comment:
Why are you expecting workflow might exist, but not a codeset?

Reply:
The idea is to make possible to rerun the script in the same environment and creating a workflow with the same name results in error (already exists).

I have changed it to delete the workflow in case it already exists.